db/migrate/20181027190121_add_domain_to_companies.rb

```diff
@@ -0,0 +1,6 @@
+class AddDomainToCompanies < ActiveRecord::Migration[5.2]
+  def change
+    add_column :companies, :domain, :string
+    add_index :companies, :domain
```

Comment:
Do we really need an index? I'm not expecting to have a huge amount of companies in the db. No problem letting it as is, though.

Reply:
uhn.. I prefer to have this. Since we will query for this when setting the subdomain, without this we will perform a full table scan every request that contains subdomains.

wdyt?

Reply:
Yes, it would indeed be a full table scan for every single request with a subdomain. Therefore it seems reasonable to keep the index. :+1: